requirements/base.txt

```diff
@@ -2,37 +2,36 @@
 # After the first "pip install -r", just run "pip freeze" and add the version
 # to each package in each requirements/*.txt.
 
-git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.0
+git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.1
 -e git+https://github.com/DemocracyClub/django-hermes.git#egg=django-hermes
-git+https://github.com/DemocracyClub/dc_django_utils.git@0.0.8
+git+https://github.com/DemocracyClub/dc_django_utils.git@0.1.1
 git+https://github.com/chrisdrackett/django-typogrify.git
 git+https://github.com/DemocracyClub/design-system.git@0.1.6
 beautifulsoup4==4.9.3
-django-braces==1.14.0
 django-cors-headers==1.1.0
 django-extensions==3.1.3
 django-localflavor==3.0.1
 django-markdown-deux==1.0.5
 django-model-utils==4.1.1
 django-ratelimit==0.6.0
-Django==2.2.24
-djangorestframework==3.12.2
+Django==3.0
```

Comment:
Now we have got the trickier part out of the, we should try to go up to 3.2 before merging?